### ISSUE TITLE
Hello, I would like to know if two GPUs can be used for training? Will the results of training differ significantly?

### ISSUE THREAD

**Issue by Rzx520** [orrzohar/PROB#22]
Hello, I would like to know if two GPUs can be used for training? Will the results of training differ significantly?

**Comment by orrzohar** [comment]
Hi @Rzx520 
Another user ran PROB with 2 GPUs instead of four and reported slight variation from the reported values. 
The variation is likely due to a difference in batch size, so a slightly lower learning rate might do the trick.

Best,
Orr

**Comment by Rzx520** [comment]
@orrzohar Thank you very much for your reply.
 May I ask if four 3090Ti cards can run out? Will the effect have a significant impact?

**Comment by orrzohar** [comment]
Hi @Rzx520,
I cannot say for certain, but there shouldn't be too much variation -- I already have three confirmed repo users that have managed to reproduce the results when training on different machines with minor variations in the final performance. 
You may need to decrease the batch size if you run out of GPU memory. Also, do not be alarmed when it crashes when running 'python test.py' - this will not affect the performance at all. Just some of the tests are too big to fit on smaller GPUs.

If training on four 3090Ti's works reasonably, could you write it here for future reference?
Best,
orr
